dbms/src/Common/RadixSort.h

```diff
@@ -179,8 +198,101 @@ struct RadixSort
     static KeyBits keyToBits(Key x) { return ext::bit_cast<KeyBits>(x); }
     static Key bitsToKey(KeyBits x) { return ext::bit_cast<Key>(x); }
 
+    static inline void insertSortInternal(Element * arr, size_t size)
```

Comment:
No motivation for `inline`.

Reply:
Removed the specifier. It does not affect performance.